core/string/ustring.cpp

```diff
@@ -4245,6 +4245,45 @@ String String::dedent() const {
 	return new_string;
 }
 
+String String::strip(const String &p_chars) const {
+	int len = length();
+	int end = len - 1;
+	int beg = 0;
+
+	if (p_chars.is_empty()) {
+		// Strip non-printable characters by default.
+		for (; beg < len; beg++) {
+			if (get(beg) > 32) {
```

Comment:
```suggestion
			if (!is_control(c) && !is_whitespace(c)) {
```
This won't cover all non-printable characters, by we at least want to strip basic control chars and all types of spaces.